catalog/core/catalog-core-ftp/src/main/java/ddf/catalog/core/ftp/FtpServerStarter.java

```diff
@@ -36,8 +40,16 @@
 
     private static final String DEFAULT_LISTENER = "default";
 
+    private static final String WANT_AUTH = "WANT";
```

Comment:
I agree that it would be nice to have something in system.properties to change both properties at the same time while still being able to remove that variable and make them have different values if needed.